deployments/kubernetes/sms-gateway-server/templates/deployment.yaml

```
@@ -0,0 +1,100 @@
+          livenessProbe:
+            {{- toYaml . | nindent 12 }}
+          {{- end }}
+          {{- with .Values.readinessProbe }}
+          readinessProbe:
+            {{- toYaml . | nindent 12 }}
+          {{- end }}
+          {{- with .Values.resources }}
+          resources:
+            {{- toYaml . | nindent 12 }}
+          {{- end }}
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)